alembic/versions/b05b6c4d1a17_spectrum_type.py

```diff
@@ -0,0 +1,28 @@
+"""spectrum type
+
+Revision ID: b05b6c4d1a17
+Revises: 171e9ea18885
+Create Date: 2021-09-29 13:55:31.409866
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b05b6c4d1a17'
+down_revision = '171e9ea18885'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('spectra', sa.Column('type', sa.String(), nullable=True))
```

Comment:
Do we want to back-populate existing rows with a sensible default?

Reply:
Yes! but I am not sure how to do that...

Reply:
Here's an example: https://github.com/skyportal/skyportal/blob/master/alembic/versions/5efdda259fd0_add_role_id_to_invitations_table.py#L24-L36

Reply:
Did you apply this migration locally to ensure that it worked to back-populate existing rows?

Reply:
Yes it now works locally with old spectra getting automatically populated as "source".